llvm/lib/Target/EVM/EVMAsmPrinter.cpp

```diff
@@ -70,12 +76,43 @@ void EVMAsmPrinter::emitFunctionEntryLabel() {
 void EVMAsmPrinter::emitInstruction(const MachineInstr *MI) {
   EVMMCInstLower MCInstLowering(OutContext, *this, VRegMapping,
                                 MF->getRegInfo());
-
   MCInst TmpInst;
   MCInstLowering.Lower(MI, TmpInst);
   EmitToStreamer(*OutStreamer, TmpInst);
 }
 
+bool EVMAsmPrinter::isBlockOnlyReachableByFallthrough(
```

Comment:
Can we add a test?

Reply:
I added a test. Please, note I've changed the implementation. Not it just always returns false for simplicity. I do not sees any drawbacks here. I moved instruction stackification to the early stage (EVMStackify), so now it's actually not possible to analyze BB terminators here (and this has no much sense).